### PR TITLE
Add support for Romanian postcodes

### PR DESCRIPTION
**Pull request by radubogdan**

### CODE REVIEW

File: lib/going_postal.rb [hunk 1]
```diff
@@ -329,6 +329,11 @@ def format_fr_postcode(string)
     format_de_postcode(string)
   end
   
+  def format_ro_postcode(string)
+    string = string.to_s.upcase.delete(" \t\r\n")
```
Comment (timblair) [inline lib/going_postal.rb]: Given that Romanian postcodes are all numeric, could you drop the call to `#upcase` in here?

File: lib/going_postal.rb [hunk 2]
```diff
@@ -33,7 +33,7 @@
 # Currently supported countries are United Kingdom (GB, UK), United States
 # (US), Canada (CA), Australia (AU), New Zeland (NZ), South Africa (ZA), The
 # Netherlands (NL), Sweden (SE), Belgium (BE), Austria (AT), Norway (NO),
-# Germany (DE), Luxembourg (LU), Denmark (DE) and France (FR).
+# Germany (DE), Luxembourg (LU), Denmark (DE), France (FR) and Romania(RO).
```
Comment (timblair) [inline lib/going_postal.rb]: I'm being slightly picky here, but could you add a space between `Romania` and `(RO)`, please?

**Comment by timblair** [comment]
Hey @radubogdan: thanks for the PR!  I've left a couple of very minor comments: once those are sorted I'll happy merge this in.

**Comment by radubogdan** [comment]
Sure @timblair. Thank you!

**Comment by radubogdan** [comment]
Any updates @timblair ?

**Comment by timblair** [comment]
Oh, sorry @radubogdan, I hadn't seen the update.  Thanks for making that change: I'll merge this now :+1:

**Comment by timblair** [comment]
`0.1.6` (which includes this change) is now available on rubygems.org.

**Pull request merged**